conda/exceptions.py

```diff
@@ -1297,16 +1297,31 @@ def __init__(self, msg: str, *args, **kwargs):
 
 
 class EnvironmentSpecPluginNotDetected(SpecNotFound):
-    def __init__(self, name: str, plugin_names: Iterable[str], *args, **kwargs):
+    def __init__(
+        self,
+        name: str,
+        plugin_names: Iterable[str],
+        autodetect_disabled_plugins: Iterable[str] = [],
+        *args,
+        **kwargs,
+    ):
         self.name = name
         msg = dals(
             f"""
-            Environment at {name} is not readable by any installed
-            environment specifier plugins.
+            Environment at {name} is not readable by any installed environment specifier plugins.
+
+            Available plugins: {dashlist(plugin_names, 16)}
 
-            Available plugins: {dashlist(plugin_names, 4)}
             """
         )
+        if len(autodetect_disabled_plugins) > 0:
+            msg += dals(
+                """
+                Found compatible plugins but they must be explicitly selected.
+                Request conda to use these plugins by providing
+                the cli argument `--environment-spec PLUGIN_NAME`:
```

Comment:
Ami I missing something or is this argument not implemented yet?

Reply:
This was implemented in another PR. If you pull this one down and try it out it should all work as described